pkgs/development/tools/ldid/default.nix

```diff
@@ -0,0 +1,70 @@
+{ lib
+, stdenv
+, callPackage
+, fetchgit
+, libplist
+, libxml2
+, openssl_1_1
+, CoreFoundation
+, Security
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ldid";
+  version = "2.1.5";
+
+  src = fetchgit {
+    url = "git://git.saurik.com/ldid.git";
```

Comment:
no https.... 

ERR_SSL_VERSION_OR_CIPHER_MISMATCH

Reply:
https://git.saurik.com seems to be using TLS 1.1, which still works on Safari. git://git.saurik.com/ldid.git should be unaffected though.